spec/src/main/asciidoc/ServerPages.adoc

```diff
@@ -11003,7 +11003,7 @@ Jakarta Server Pages specification. This appendix is non-normative.
 
 === Changes between JSP 3.0 and JSR 245
 
-* The API has moved from the `javax.servlet.jsp` package to the
+* The API has moved from the `jakarta.servlet.jsp` package to the
```

Comment:
This doesn't seem entirely correct ;)

Reply:
Whoops :)  Will change that back.  Tried to catch statements like those, but this one slipped through the cracks.  Thanks!

Reply:
Fixed